tests/coverage_llvm-cov/read.me

```diff
@@ -0,0 +1,17 @@
+# docs:
+ LLVM-COV: https://llvm.org/docs/CommandGuide/llvm-cov.html
+ LCOV:     http://ltp.sourceforge.net/coverage/lcov/lcov.1.php
+# source code:
+ LLVM-COV: git clone http://llvm.org/git/llvm.git
+ LCOV:     http://ltp.sourceforge.net/coverage/lcov.php
+# install using apt-get:
+ LLVM-COV: when installing using apt-get install llvm-<NUMBER>, llvm-cov will be installed with a suffix of the version number as well.
+ $ apt-get install llvm-<NUMBER>
+ LCOV:
+ $ apt-get install lcov
+
+# How to use
+ Run script:
```

Comment:
Create a build folder, "cd" into it and execute the script:
$ facelift/tests/coverage_llvm-cov/test_llvm-cov.sh